### DISCUSSION TITLE
Comment not processed while in Preview mode

### DISCUSSION THREAD

**Discussion by Elliria** [community/community#56639] [Discussions]
### Select Topic Area

Bug

### Body

When writing a comment in a Discussion or anywhere on GitHub where a comment can be written, if the **Preview** tab is active when you click the **Comment** button, nothing visibly happens except that the **Comment** button dims and becomes unclickable. Your comment text will still be in the **Preview** tab and no new comment will have been added to the conversation above. If you then click on the **Write** tab, your typed text will be cleared from the **Preview** tab and your comment will be added to the conversation above. There is no issue if the **Write** tab is active when you click the **Comment** button.

**Comment by dakshithadissanayaka** [comment]
The behavior you're describing is indeed a bug in the GitHub commenting system. When you click the "Comment" button while the "Preview" tab is active, the comment is not actually posted, and the button becomes unclickable. This is likely due to an error in the handling of the "Preview" mode.

The expected behavior is that the comment should be posted regardless of which tab is active, "Write" or "Preview". The "Preview" mode should simply provide a way to preview the comment before it's submitted, but it should not interfere with the actual posting of the comment.

I recommend reporting this bug to GitHub's support team so that they can investigate and fix it. In the meantime, you can work around the bug by switching to the "Write" tab before clicking the "Comment" button.

**Comment by Elliria** [comment]
Thank you for verifying the bug, @dakshithadissanayaka. Do you happen to have instructions for exactly how I would go about reporting this bug to GitHub's support team? I looked up how to file a bug report with GitHub and this is where it brought me, so I thought I was filling out a bug report by opening this discussion. Any help would be appreciated.

**Comment by Elliria** [comment]
Thank you for letting me know it's a known issue. Do you have a link to the issue so I can keep an eye on it (and learn where actual GitHub issues are located for future reference)?

**Comment by PavanLuca** [comment]
I see the same issue but in my case the comments are published anyway, even if I was on `Preview` before clicking `Comment` the comment is effectively published, GitHub just doesn't close the preview once published, that's it.
![Screenshot 2023-12-28 at 15-20-26 Broken wielded item loaded behind the character · Issue #13710 · minetest_minetest](https://github.com/community/community/assets/123581658/b21e25bc-fed0-4326-be75-49ba32b0e5e1)

**Comment by Elliria** [comment]
Same here. And if the **Preview** tab is active when you click the **Comment** button, nothing visibly happens except that the **Comment** button dims and becomes unclickable. Your comment text will still be in the **Preview** tab and no new comment will have been visibly added to the conversation above, even though, as you pointed out, the comment will actually have been added to the conversation.

**Reply by PavanLuca** [reply]
Yes, the button is dimmed and becomes unclickable, too. Luckily the comment is posted. As I said, what we see is just a “glitch”, an appearance, otherwise GitHub team would receive a ton of reports for missing comments.

**Reply by Elliria** [reply]
Hopefully that means that it will be a simple issue to fix.
